console/module/api/src/main/java/org/eclipse/kapua/app/console/module/api/client/ui/grid/KapuaGrid.java

```diff
@@ -37,6 +37,6 @@ public KapuaGrid(ListStore<M> store, ColumnModel cm) {
         GridView gridView = getView();
         gridView.setAutoFill(true);
         gridView.setForceFit(true);
-        gridView.setSortingEnabled(false);
+        gridView.setSortingEnabled(true);
```

Comment:
Ok for this line, but we probably also want `gridView.setRemoteSorting(true)` so that any sort triggers a new query to the service. Also be sure to test that the behavior is correct in all the grids after this.

Reply:
Actually, I see that the backend services needs to be changed in order to support remote sorting. I'll do this ASAP.